### DISCUSSION TITLE
DDF request

### DISCUSSION THREAD

**Discussion by huliz** [community/community#123336] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, I tried to respond that I'd like to try your new DDF for Aqara curtain device but the post is blocked for comments for me

**Comment by ebndev** [comment]
Hi @huliz,

Thanks for being a part of the GitHub Community, we're glad you're here!

The Profile category is for conversations related to your GitHub profile page. I have moved this post to General, however, it looks like you're looking for support from a specific project. You might want to try asking for help somewhere that focuses on this project, such as opening an issue in the project's repository or seeing if they have a support page.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!

**Comment by ebndev** [comment]
Hi again @huliz!

If you don't mind, would you mind letting us know how you initially ended up in the [Profile category](https://github.com/orgs/community/discussions/categories/profile?discussions_q=is%3Aopen+category%3AProfile+) when creating your discussion and providing the link to the project's repository? We are trying to investigate if there is a bug or link that may be re-directing users from the project's Discussions board, if that’s where you intended to post.

Any insight would be helpful, thanks!

**Comment by huliz** [comment]
I tried to send a message to a user who wrote me on an issue that was
closed for new comments. I was never able to contact the user...

**Reply by ebndev** [reply]
I see! Did you manually go to GitHub Discussions or were you re-directed from their profile / the issue?

**Comment by huliz** [comment]
I don't remember. I clicked on the profile, there was no "send a DM" or
something like that, so I probably chose something that resembled it

**Reply by ebndev** [reply]
Found the issue! Thanks again for your time and being a part of the GitHub Community 😄

**Comment by huliz** [comment]
cool, so, is there a way I can send a message to the user?

**Reply by ebndev** [reply]
No, there is no way to DM users directly. Are you able to open a new issue in their repo? Otherwise you would have to find an external way to contact them (support page, social media, etc.)
